### ISSUE TITLE
Not following griefprevention etc.

### ISSUE THREAD

**Issue by KasperFranz** [McJty/interactionwheel#5]
The rotate, fetch from container and equal to container  works even when it shouldn't.

the dump to container did not work

This is all done on a vanilla chest using Sponge and GriefPrevention on a Beyond 1.0.1 server

**Comment by KasperFranz** [comment]
The right method should ofc. be that you can't take from/deposit to a claimed chest

**Comment by McJty** [comment]
How can I even do that? That is not up to this mod to handle as far as I know

**Comment by KasperFranz** [comment]
I have no idea - I have never done anything related to forge :p 

There is no events fired so no way to catch and cancel it

**Comment by KasperFranz** [comment]
Do you have an update on this :)?

**Comment by McJty** [comment]
I don't think this is something that has to be fixed on my side. It is up to the grief prevention tools to do that as far as I know. I wouldn't even know how to fix this. Maybe report to whatever project you're using for grief prevention

**Comment by gabizou** [comment]
@McJty Just throwing an idea out here, but I believe it's possible for Sponge to provide protection plugins the ability to "cancel" the interactions (in this case, extracting or opening inventories) by doing MinecraftServer.isBlockProtected(World, BlockPos, EntityPlayer). Since Sponge will throw an event for plugins to interact with the block, I do believe this is the ultimate thing we can do to achieve compatibility with the mod.

It looks like all you have to do really is just perform the check prior to sending or opening inventories for tile entities or `IItemHandler`s etc.
